jsonenums.go

```diff
@@ -75,7 +75,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/campoy/jsonenums/parser"
+	"github.com/adfin/jsonenums/parser"
```

Comment:
hah, no

check this out: http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html